source/deployment/team-channel-management.rst

```diff
@@ -0,0 +1,106 @@
+Managing Team and Channel Members
```

Comment:
Is this E20 only currently?

Reply:
Good question. @thefactremains - can you confirm?

Reply:
It's E20.

Reply:
Updated.